openlibrary/plugins/upstream/models.py

```diff
@@ -411,7 +411,7 @@ def wp_citation_fields(self):
         Builds a wikipedia citation as defined by http://en.wikipedia.org/wiki/Template:Cite#Citing_books
         """
         result = {
-            "title": self.works[0].title.replace("[", "&#91").replace("]", "&#93"),
```

Comment:
I'm not sure the consequences of removing `replace("[", "&#91").replace("]", "&#93")` -- it looks messy so I'm happy to see it go. It looks like it is meant to make the text websafe, but if so it should be done in one location

Reply:
So I'm assuming need to revert on the change?

Reply:
oh no, the `self.title` was correct as discussed, but you also removed character replacements, which was an additional change.

`self.title.replace("[", "&#91").replace("]", "&#93"),`   is what we want -- replace the work title with the edition title, and leave the required character escaping as is.

Reply:
Alright so final code
```
"title":self.title
```
```
"edition": self.title.replace("[", "&#91").replace("]", "&#93")
```

Reply:
@hornc does this look fine. Apologies for the misunderstanding.